docs/UserGuide.md

```diff
@@ -531,7 +533,7 @@ Example: `98739283`
 Example: `e/e3029834@u.nus.edu`
 
 ### `nat/NATIONALITY`
-`NATIONALITY` is only valid if it is one of the nationality listed in the [list of nationalities CSV file](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/664133/CH_Nationality_List_20171130_v1.csv/preview) by GOV.UK.
+`NATIONALITY` is only valid if it is one of the nationality listed in this [list of nationalities TXT file](https://gist.github.com/marijn/274449).
```

Comment:
Perhaps we can copy paste the file into our team repo, to protect UG in the rare case this file is removed from his repo.

Reply:
Yes it's alrdy in our repo under `src/main/resources/data/nationalities.txt`